src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -95,4 +110,62 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
   }
+
+  public List<ScheduledInterviewRequest> scheduledInterviewsToRequestObjects(
+      List<ScheduledInterview> scheduledInterviews, String timeZoneIdString) {
+    ZoneId timeZoneId;
+    if (timeZoneIdString == null) {
```

Comment:
Q: when would this ever be null?

Reply:
I was null for my tests but I just added a parameter in the tests and removed this.